TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/BoundAndroidMovementService.java

```diff
@@ -0,0 +1,59 @@
+package com.novoda.tpbot.bot;
+
+import android.app.Service;
+import android.content.Intent;
+import android.os.Binder;
+import android.os.IBinder;
+import android.support.annotation.Nullable;
+
+public class BoundAndroidMovementService extends Service {
```

Comment:
Want to highlight that this is the `Bound` variety of `Service` to avoid confusion.